contrib/compose/README.md

```diff
@@ -1,23 +1,147 @@
 # Docker Compose V1 + V2 registry
 
-This compose configuration will setup a v1 and v2 registry behind an nginx
-proxy. By default the combined registry may be accessed at localhost:5000.
-This registry does not support pushing images to v2 and pull from v1. Clients
-from before 1.6 will be configured to use the v1 registry, and newer clients
-will use the v2 registry.
-
-## Prerequisites
-Install [docker-compose](https://github.com/docker/compose)
-
-## How to run
-```
-$ docker-compose up
-```
-
-## How to push images
-From a local project directory with Dockerfile
-```
-$ docker build -t localhost:5000/myimage .
-$ docker push localhost:5000/myimage
-```
+This compose configuration configures a `v1` and `v2` registry behind an `nginx`
+proxy. By default, you can access the combined registry at `localhost:5000`.
+
+The configuration does not support pushing images to `v2` and pulling from `v1`.
+If a `docker` client has a version less than 1.6, Nginx will route its requests
+to the 1.0 registry. Requests from newer clients will route to the 2.0 registry.
+
+### Install Docker Compose
+
+1. Open a new terminal on the host with your `distribution` source.
+
+2. Get the `docker-compose` binary.
+
+		$ sudo wget https://github.com/docker/compose/releases/download/1.1.0/docker-compose-`uname  -s`-`uname -m` -O /usr/local/bin/docker-compose
+
+	This command installs the binary in the `/usr/local/bin` directory. 
+	
+3. Add executable permissions to the binary.
+
+		$  sudo chmod +x /usr/local/bin/docker-compose
+		
+## Build and run with Compose
+	
+1. In your terminal, navigate to the `distribution/contrib/compose` directory
+
+	This directory includes a single `docker-compose.yml` configuration.
+	
+		nginx:
+			build: "nginx"
+			ports:
+				- "5000:5000"
+			links:
+				- registryv1:registryv1
+				- registryv2:registryv2
+		registryv1:
+			image: registry
+			ports:
+				- "5000"
+		registryv2:
+			build: "../../"
+			ports:
+				- "5000"
+
+ 		This configuration builds a new `nginx` image as specified by the
```

Comment:
This indentation puts the paragraph in the configuration grey box.  I'm not sure if it's intentional, but you lost the `` italic effect.